labs-ios-starter/Model/Apollo/Topic/ContextObject.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ContextQuestion.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/8/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+/// Used to determine a Question's context. so named to avoid collision with Swift.Context
+struct ContextObject: Codable {
+    
+    enum CodingKeys: String, CodingKey {
+        case id
+        case title = "contextoption"
+    }
+
+    var id: Int
+    var title: String
+
```

Comment:
Should remove the extra line here. 

We should consider using a linter with a custom rule set once the applications MVP is in place.